routes/api.php

```diff
@@ -14,6 +15,5 @@
 |
 */
 
-Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
-    return $request->user();
-});
+Route::apiResource('products',ProductController::class);
```

Comment:
make product controller index and show public available and hide the rest behind authentication to give access to create/update/delete product only to admin

Reply:
Ok. will do it after Implement  Auth(JWT)  to the project(just try to put things one by one).